examples/time-log-kopia-repository-server/README.md

```diff
@@ -0,0 +1,233 @@
+# Walkthrough of Time Log
+
+This is an example of using Kanister to protect a simple application called Time Log. This application is contrived but useful for demonstrating Kanister's features. Every second it appends the current time to a log file.
+
+Note: This application is used in the Kanister tutorial, for a detailed walkthrough click [here](https://docs.kanister.io/tutorial.html#tutorial).
+
+### 1. Deploy the Application
+
+The following command deploys the example Time Log application in `time-log` namespace:
+
+```bash
+$ kubectl create namespace time-log
+```
+
+```bash
+# Create a deployment whose log we'll ship to s3 using kopia repository server
+$ kubectl apply -f ./examples/time-log/time-logger-deployment.yaml -n time-log
+deployment "time-logger" created
+```
```

Comment:
```suggestion
```bash
$ kubectl create namespace time-log

# Create a deployment whose log we'll ship to s3 using Kanister functions (that would eventually use Kopia repository server)
$ kubectl apply -f ./examples/time-log/time-logger-deployment.yaml -n time-log
deployment "time-logger" created
```
```